frontend/src/app/survey/survey-tab-view/survey-tab-view.component.html

```diff
@@ -13,7 +13,7 @@
             <app-abstract-survey-table [filter]="{ didParticipateIn: true, isUpcoming: true }"></app-abstract-survey-table>
         </mat-tab>
         <mat-tab label='{{ "survey.tabs.open_surveys" | translate }}'>
-            <app-abstract-survey-table [filter]="{ isUpcoming: true }"></app-abstract-survey-table>
+            <app-abstract-survey-table [filter]="{ isUpcoming: true, isCompleted: false }"></app-abstract-survey-table>
```

Comment:
You must remove `isUpcoming: true`. That is the filter to only show completed surveys which have a selected time slot in the future.